orienteer-core/src/main/java/org/orienteer/core/component/table/OPropertyValueColumn.java

```diff
@@ -50,4 +55,16 @@ public OPropertyValueColumn(String sortProperty, IModel<OProperty> criteryModel,
 		return new OPropertyNamingModel(getCriteryModel());
 	}
 
+	@SuppressWarnings("unchecked")
+	@Override
+	public Component getFilter(final String componentId, FilterForm<?> form) {
+		final IModel<OProperty> propertyModel = getCriteryModel();
+		UIVisualizersRegistry registry = OrienteerWebApplication.lookupApplication().getUIVisualizersRegistry();
+		IVisualizer visualizer = registry.getComponentFactory(propertyModel.getObject().getType(), "default");
```

Comment:
Here you are taking default visualizer. Technically it's not good: you should first use visualizer defined on the property and if it's returning null for filter - get filter from default visualizer.